### PR TITLE
Valentine's Day Animation

### PR DESCRIPTION
**Pull request by alvaro-ricotta**
[Apex_1707713945494.webm](https://github.com/bobanetwork/gateway/assets/81116391/a8aadf86-23ec-4c5e-bc14-0dad1f57fd31)

Backlog: 

- An animation was added for Valentine's Day.
- A date filter was implemented so that the animation is visible only on February 14th.
- Small details of the scroll on large screens were corrected.

### CODE REVIEW

File: src/util/animations/index.tsx
Comment (sk-enya) [inline src/util/animations/index.tsx:1]: This is the react components so should has to be go into the components folder not in the util.

File: src/util/animations/index.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      <Realistic onInit={onInit} decorateOptions={heartsOptions} />
+    </CanvaContainer>
+  )
+}
+
+export const ValentinesDayBanner = () => {
+  const [isValentinesDay, setIsValentinesDay] = useState(false)
+
+  useEffect(() => {
+    const today = dayjs()
```
Comment (sk-enya) [inline src/util/animations/index.tsx:79]: Let's have more correct logic here! 

this only needs to be for 14th or should be visible for week!

**Review by sk-enya — CHANGES_REQUESTED** [review]

**Review by wsdt — APPROVED** [review]
agree with Sahil, looks good otherwise!

File: src/util/animations/index.tsx (hunk 1 above)
Comment (alvaro-ricotta) [reply src/util/animations/index.tsx]: Already ask about it, but nobody answer this question yet.

**Review by sk-enya — APPROVED** [review]

**Pull request merged**